pkg/northbound/admin/admin.go

```diff
@@ -175,3 +177,62 @@ func (s Server) RollbackTransaction(ctx context.Context, req *admin.RollbackRequ
 	}
 	return nil, ctx.Err()
 }
+
+// LeafSelectionQuery selects values allowable for leaf.
+func (s Server) LeafSelectionQuery(ctx context.Context, req *admin.LeafSelectionQueryRequest) (*admin.LeafSelectionQueryResponse, error) {
+	log.Debugf("Received LeafSelectionQuery %+v", req)
+	logContext(ctx, "LeafSelectionQuery()")
+	if req == nil {
+		return nil, errors.NewInvalid("request is empty")
```

Comment:
You need to use errors.Status here to return error. 
like this https://github.com/onosproject/onos-config/blob/master/pkg/northbound/admin/admin.go#L128

Reply:
done